src/App.js

```diff
@@ -1,12 +1,13 @@
 import './App.css';
-import Calculator from './components/Calculator';
-import Quote from './components/Quotes';
+// import Calculator from './components/Calculator';
+import Header from './components/Header/Header';
+// import Quote from './components/Quotes';
 
 function App() {
   return (
     <>
-      <Calculator />
-      <Quote />
+      {/* <Calculator /> */}
+      <Header />
```

Comment:
- [ ] Please remove the commented code from the files, It's not good practice to leave commented code in files.